src/main/java/com/wynntils/hades/protocol/packets/server/HSPacketUpdateMutual.java

```diff
@@ -10,29 +10,42 @@
  * Used for updating mutual friends/party users their status.
  */
 public class HSPacketUpdateMutual implements HadesPacket<IHadesClientAdapter> {
-
     UUID user;
+    String name;
     double x, y, z;
     double health, mana;
     double maxHealth, maxMana;
+    String world;
```

Comment:
Sorry forgot about this, world data also seems redundant since update mutual should only be sent for players that are on the same world

Reply:
Why is that the case? Why can't this work on any server?

Reply:
Because if you have a friend list with 500 players and all of them are suddenly online you would receive a bunch of useless data. The mutual update should only be used to display players on the map, perhaps renaming so it is more clear is a good strategy!

Reply:
Okay, sure.